### DISCUSSION TITLE
Why is CapSolver so expensive (Part 2.)

### DISCUSSION THREAD

**Discussion by BeeperMiniSourceCode** [david96182/ninjemail#10] [General]
When I go to market it says that it's $200-300. I don't see a way to buy it for $1.6/1000 requests $0.6/1000 images. I only see where it says market. How do/did you buy it for that much?

**Comment by BeeperMiniSourceCode** [comment]
![image](https://github.com/david96182/ninjemail/assets/154485916/2e5f8c35-e99c-426e-bcc1-f7601d228a00)

**Comment by BeeperMiniSourceCode** [comment]
Ohh I see

**Comment by BeeperMiniSourceCode** [comment]
@david96182 How do I buy it cheap?

**Reply by david96182** [reply]
Create an account and go to this page https://dashboard.capsolver.com/dashboard/checkout. The last option in the 'Add credits' section allows you to put the desired amount you want to buy, minimum is $6.

**Reply by BeeperMiniSourceCode** [reply]
Ok what do I do after. How do I buy requests?

**Reply by david96182** [reply]
Having credits in your account is enough. If you want to know how much that credits will last depending on your usage check the pricing page.

**Reply by BeeperMiniSourceCode** [reply]
I don't get it. I have the credits in my account. What do I do now? Do I just run ninjemail and i'll automatically use it for FunCaptcha?

**Reply by david96182** [reply]
Yes, that's it.

**Reply by BeeperMiniSourceCode** [reply]
Ok Love you <3
